tools/RAiDER/models/weatherModel.py

```diff
@@ -439,10 +439,16 @@ def checkContainment(self: weatherModel,
            and False otherwise.
         """
         ymin_input, ymax_input, xmin_input, xmax_input = ll_bounds
-        input_box = box(xmin_input, ymin_input, xmax_input, ymax_input)
+        input_box   = box(xmin_input, ymin_input, xmax_input, ymax_input)
         xmin, ymin, xmax, ymax = self.bbox
         weather_model_box = box(xmin, ymin, xmax, ymax)
 
+        world_box  = box(-180, -90, 180, 90)
```

Comment:
I don't think you need the if-statement, just transform everytime. We should just make sure to add a small buffer to the download region

Reply:
Do all the weather models have a projection though?

Reply:
All models should come with projections.

Reply:
I think we should still have the check that the user input is in geocoordinates before converting

Reply:
Makes sense.